.gitignore

```diff
@@ -1 +1,2 @@
 target/
+/.vs
```

Comment:
What is this change? I probably need you to remove it.

Reply:
This is related to a folder created by Visual Studio.
Adding this exclusion the project can also be edited with Visual Studio.